grammar.js

```diff
@@ -413,10 +412,10 @@ module.exports = grammar({
       optional(',')
     ),
 
-    expression_list: $ => seq(
+    expression_list: $ => prec.right(seq(
       commaSep1($._expression),
       optional(',')
-    ),
+    )),
```

Comment:
Update on this precedence. This disambiguates between `expression_list` and `yield expression_list`. There is likely another way to do this, in case we're not happy with the `prec.right`.